grakn/service/Session/util/ResponseReader.py

```diff
@@ -27,19 +28,15 @@
 class ResponseReader(object):
     
     @staticmethod
-    def query(tx_service, grpc_query_iter):
-        iterator_id = grpc_query_iter.id
-        return ResponseIterator(tx_service,
-                                iterator_id,
-                                lambda tx_serv, iterate_res: AnswerConverter.convert(tx_serv, iterate_res.query_iter_res.answer))
-
+    def query(tx_service):
```

Comment:
Ok so i think this method's naming is the incorrect - this is really the `ReasonerReader.from_iterate_response` or something right? `query` is too generic and doesn't carry any meaning anymore